templates/batch-llm/README.md

```diff
@@ -1,326 +1,193 @@
-# LLM offline batch inference with Ray Data and vLLM
+# LLM offline batch inference with RayLLM-Batch APIs
 
 **⏱️ Time to complete**: 10 min
 
-This template shows you how to:
-1. Read in data from in-memory samples or files on cloud storage. 
-2. Use Ray Data and vLLM to run batch inference of a LLM.
-3. Write the inference outputs to cloud storage.
 
-For a Python script version of the code in this workspace template, refer to `main.py`.
+<!-- TODO: add a link for the RayLLM-Batch API reference -->
+This template shows you how to run batch inference for LLMs using RayLLM-Batch.
```

Comment:
Since we don't want to run anyscale submit job for now (because the log outputs are just not properly handled yet) so I didn't add it. 

I was actually planning to remove the main.py - good catch. I am open to have the main.py as well, no strong preference.

Reply:
sounds good to remove, yeah the main point of the `main.py` was to run as a job, and also allow the user to run the notebook code as one continuous script.